packages/grafana-ui/src/components/uPlot/geometries/types.ts

```diff
@@ -19,6 +20,13 @@ export interface AreaProps {
   color: string;
 }
 
+export enum AxisSide {
+  Top, // 0
+  Right, // 1
+  Bottom, // 2
+  Left, // 3
```

Comment:
```suggestion
  Top = 0,
  Right = 1,
  Bottom = 2,
  Left = 3,
```
Or are there any reasons not to define it this way?

Reply:
Maybe also add a code comment about the numeric values are being matched with the ones in uPlot.